package.json

```diff
@@ -25,7 +25,9 @@
   "scripts": {
     "test": "npm run lint && mocha --recursive tests/*.spec.js -R spec && node tests/e2e.js",
     "lint": "eslint lib tests bin example.js",
-    "readme": "jsdoc2md --template doc/README.hbs lib/index.js > README.md"
+    "readme": "jsdoc2md --template doc/README.hbs lib/index.js > README.md",
+    "bump": "npm version --tag-version-prefix \"\" -m \"v%s\"",
+    "version": "versionist"
```

Comment:
Well, then it wouldn't be run as the [`npm version`](https://docs.npmjs.com/cli/version) hook anymore (which does the tagging & commit etc). Could rename it to `changelog` and make the `"version": "npm run changelog"` though. What do you think?